test/installer/scripts/omsadmin_systest.rb

```diff
@@ -296,4 +310,28 @@ def test_reconstruct_from_partial_empty_omsadmin_conf
                  "tmp directory was mistakenly created")
   end
 
+  def test_show_omsagent_proc_limit_default
+    set_omsagent_proc_limit()
+    lso = show_omsagent_proc_limit()
+    assert_match(/^-?\d+$/, lso, "The command to show what should be the default proc limit did not yield an integer.")
+  end
+
+  def test_show_omsagent_proc_limit_5000
+    set_omsagent_proc_limit(5000)
+    lso = show_omsagent_proc_limit()
+    assert_match(/^5000$/, lso, "Should have been 5000 process limit.")
+  end
+
+  def test_show_omsagent_proc_limit_two_entries
+    FileUtils.cp("#{@omsadmin_test_dir}/limits-two-settings.conf", @proc_limits_conf)
+    lso = show_omsagent_proc_limit()
+    assert_match(/^100$/, lso, "Multiple entries should still report last entry, in compliance with specification.")
+  end
+
+  def test_show_omsagent_proc_limit_no_entry
+    FileUtils.cp("#{@omsadmin_test_dir}/limits-no-settings.conf", @proc_limits_conf)
+    lso = show_omsagent_proc_limit()
+    assert_equal("", lso, "The integer associated with unlimited is -1, but we return blank for reporting.")
```

Comment:
Add a check that the "Process limit not currently set for $TEST_USER." message is printed

Reply:
Got that too.  BTW I realized I changed these in my cleanup bout last week too, so the messages try to be more grammatically clear and more correct at the same time now.  I think this will be okay, but sorry that's one more thing for you to review.